hasura/metadata/databases/default/tables/public_team.yaml

```diff
@@ -56,41 +56,36 @@ select_permissions:
         - name
         - updated_at
       filter:
-        _and:
-          - memberships:
-              team_id:
-                _eq: X-Hasura-Team-Id
-          - memberships:
-              user_id:
+        memberships:
+          _and:
+            - user_id:
                 _eq: X-Hasura-User-Id
+            - team_id:
+                _eq: X-Hasura-Team-Id
```

Comment:
@andy-t-wang @pdtfh Just found out that we need to remove this team_id check from select row check in team table for user role. Otherwise, we can't pick a list of teams for teams selector.

I think it's ok, user should be able select all the teams related with his x-hasura-user-id. Or am I missing something? 

I will push this change when I finish team selector

Reply:
yes, aligned on this